scripts/content/VACMS-15422-vet-center-outstation-copy-title.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * @file
+ * Copy Vet Center - Outstation names to field_official_name.
+ *
+ * Vet Center - Outstation now supports a common name
+ * (usually associated with place) and an official name.
+ *
+ * VACMS-15422-vet-center-outstation-copy-title.php.
+ */
+
+require_once __DIR__ . '/script-library.php';
+
+use Psr\Log\LogLevel;
+
+$revision_message = 'In a separate action, the current facility name was copied from the node title to the "Name of Vet Center - Outstation" field.';
+$sandbox = ['#finished' => 0];
+do {
+  print(va_gov_copy_vet_center_os_title($sandbox, $revision_message));
+} while ($sandbox['#finished'] < 1);
+// Node processing complete.  Call this done.
+return;
+
+/**
+ * Copy node title to field_official_name.
+ *
+ * @param array $sandbox
+ *   Modeling the structure of hook_update_n sandbox.
+ * @param string $revision_message
+ *   Text to be used in revision log message.
+ *
+ * @return string
+ *   Status message.
+ */
+function va_gov_copy_vet_center_os_title(array &$sandbox, $revision_message) {
+  $node_storage = \Drupal::entityTypeManager()->getStorage('node');
```

Comment:
```suggestion
  $node_storage = get_node_storage();
```
Minor: Existing patterns.